messaging/src/main/java/org/axonframework/messaging/DelayedMessageStream.java

```diff
@@ -148,7 +179,8 @@ public <R> CompletableFuture<R> reduce(@Nonnull R identity, @Nonnull BiFunction<
      *
      * @param <M> The type of {@link Message} contained in the {@link Entry} of this stream.
      */
-    static class Single<M extends Message<?>> extends DelayedMessageStream<M> implements MessageStream.Single<M> {
+    public static class Single<M extends Message<?>> extends DelayedMessageStream<M>
```

Comment:
You made it public, but I assume it doesn't need to be - current project compiles without that.

Reply:
Woops my mistake, will undo it.